MetroidPrimeClient.py

```diff
@@ -189,22 +189,14 @@ def __int_to_reversed_bits(value: int, bit_length: int) -> str:
 
 
 async def handle_checked_location(ctx: MetroidPrimeContext, current_inventory: dict[str, InventoryItemData]):
-    """Uses the current amount of UnknownItem1 in inventory as an indicator of which location was checked. This will break if the player collects more than one pickup without having the AP client hooked to the game and server"""
-    unknown_item1 = current_inventory["UnknownItem1"]

```

Comment:
Could you also remove the relevant patch from Container.py? There is a hefty function there that can now be stripped out that updates these values

Reply:
oh didn't know about this one. I'll edit it then